src/Microsoft.CodeAnalysis.NetAnalyzers/tests/Microsoft.CodeAnalysis.NetAnalyzers.UnitTests/Microsoft.CodeQuality.Analyzers/Maintainability/CSharpUseCrossPlatformIntrinsicsTests.MinMax.cs

```diff
@@ -0,0 +1,114 @@
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.CodeQuality.CSharp.Analyzers.Maintainability.CSharpUseCrossPlatformIntrinsicsAnalyzer,
+    Microsoft.CodeQuality.CSharp.Analyzers.Maintainability.CSharpUseCrossPlatformIntrinsicsFixer>;
+
+namespace Microsoft.CodeQuality.Analyzers.Maintainability.UnitTests
+{
+    using static UseCrossPlatformIntrinsicsAnalyzer;
+
+    public partial class CSharpUseCrossPlatformIntrinsicsTests
+    {
+        [Theory(Skip = "MaxNative/MinNative are .NET 9+ APIs not available in test framework")]
+        [InlineData("float", "Sse.Max")]
+        [InlineData("float", "Sse.Min")]
+        public async Task Fixer_MinMaxSseAsync(string type, string method)
+        {
+            var ruleKind = method.Contains("Max") ? RuleKind.MaxNative : RuleKind.MinNative;
+            var expectedMethod = method.Contains("Max") ? "MaxNative" : "MinNative";
+
+            // lang=C#-test
+            string testCode = $$"""
+                using System;
+                using System.Runtime.Intrinsics;
+                using System.Runtime.Intrinsics.X86;
+
+                class C
+                {
+                    Vector128<{{type}}> M(Vector128<{{type}}> x, Vector128<{{type}}> y) => {|#1:{{method}}(x, y)|};
+                }
+                """;
+
+            // lang=C#-test
+            string fixedCode = $$"""
+                using System;
+                using System.Runtime.Intrinsics;
+                using System.Runtime.Intrinsics.X86;
+
+                class C
+                {
+                    Vector128<{{type}}> M(Vector128<{{type}}> x, Vector128<{{type}}> y) => Vector128.{{expectedMethod}}(x, y);
+                }
+                """;
+
+            await new VerifyCS.Test
+            {
+                TestCode = testCode,
+                ExpectedDiagnostics = {
+                    VerifyCS.Diagnostic(Rules[(int)ruleKind]).WithLocation(1),
+                },
+                FixedCode = fixedCode,
+                ReferenceAssemblies = ReferenceAssemblies.Net.Net80
+            }.RunAsync();
+        }
+
+        [Theory]
+        [InlineData("byte", "AdvSimd.Max")]
+        [InlineData("byte", "AdvSimd.Min")]
+        [InlineData("sbyte", "AdvSimd.Max")]
+        [InlineData("sbyte", "AdvSimd.Min")]
+        [InlineData("short", "AdvSimd.Max")]
+        [InlineData("short", "AdvSimd.Min")]
+        [InlineData("ushort", "AdvSimd.Max")]
+        [InlineData("ushort", "AdvSimd.Min")]
+        [InlineData("int", "AdvSimd.Max")]
+        [InlineData("int", "AdvSimd.Min")]
+        [InlineData("uint", "AdvSimd.Max")]
+        [InlineData("uint", "AdvSimd.Min")]
+        [InlineData("float", "AdvSimd.Max")]
+        [InlineData("float", "AdvSimd.Min")]
+        public async Task Fixer_MinMaxAdvSimdAsync(string type, string method)
```

Comment:
This is only covering tests for Vector64. I'd expect tests for Vector128 to also exist.